examples/wireguard-with-cilium/README.md

```diff
@@ -0,0 +1,127 @@
+# Transparent Encryption with Cilium and Wirguard
+
+This example shows how to provision an EKS cluster with:
+- Managed node group based on Bottlerocket AMI
+- Cilium configured in CNI chaining mode with VPC CNI and with Wiregaurd transparent encryption enabled
+
+## Reference Documentation:
+
```

Comment:
What about adding the link to Aaron's blog post?

Reply:
Its not published yet :)

Reply:
whoops!